src/components/try-hcp-callout/content.ts

```diff
@@ -5,10 +5,17 @@
 
 import { TryHcpCalloutProps } from 'components/try-hcp-callout/types'
 import { ProductSlugWithContent } from './types'
+import boundarySvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-boundary.svg?include'
+import consulSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-consul.svg?include'
+import packerSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-packer.svg?include'
+import terraformSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-terraform.svg?include'
+import vagrantSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-vagrant.svg?include'
+import vaultSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-vault.svg?include'
+import waypointSvg from './components/try-hcp-callout/img/try-hcp-callout-ui-mock-waypoint.svg?include'
```

Comment:
If the increase in bundle size is in fact due to this change, I wonder if either there are [some optimizations](https://jakearchibald.github.io/svgomg/) that can be made to the SVG or if we can use a different file format.